django_migration_zero/models.py

```diff
@@ -14,7 +14,7 @@ class MigrationZeroConfiguration(models.Model):
     )
     migration_date = models.DateField(_("Migration date"), null=True, blank=True)
 
-    objects = MigrationZeroConfigurationManager()
+    objects = MigrationZeroConfigurationQuerySet.as_manager()
```

Comment:
question: why are you changing the manger setup? I don't see any reason why to do so in the other changes 🙂

Reply:
I changed it to a QuerySet class so it's possible to use chained methods - the `fetch_singleton` on the top of the `select_for_update` - so I needed to adjust the syntax here [[ref](https://docs.djangoproject.com/en/5.0/topics/db/managers/#creating-a-manager-with-queryset-methods)].

Reply:
Ah, now I get it! I like the idea but there is a convention to make a queryset method always return a queryset. That's why I put the `fetch_singleton()` in a manager method.

Do you have an idea on how we could obey by the rule and still get this merged? 🤔

Reply:
Got it! It worked fine on the manager by moving the `select_for_update` inside the `fetch_singleton` (last commit).

Reply:
Awesome ❤️